jaxrs-api/src/main/java/jakarta/ws/rs/core/Response.java

```diff
@@ -232,19 +232,18 @@ protected Response() {
     public abstract boolean hasEntity();
 
     /**
-     * Buffer the message entity data.
+     * Buffer the message entity input stream.
      * <p>
-     * In case the message entity is backed by an unconsumed entity input stream, all the bytes of the original entity input
-     * stream are read and stored in a local buffer. The original entity input stream is consumed and automatically closed
-     * as part of the operation and the method returns {@code true}.
+     * In case the message entity is backed by an input stream not consumed at all, all the bytes of that input
+     * stream are read (i.e until <code>inputStream.read() == -1</code>) and stored in a local buffer. That input
```

Comment:
Overall, this looks like a step in the right direction. However, I think the phrase "not consumed at all" is a bit vague. If we mean, "not a single byte from the input stream has been read", we should be more explicit IMO. This phrase is used multiple times.

Reply:
@NicoNes Could you review and respond to the comment?

Reply:
@spericas I agree we can be more exeplicit.  
What about saying " In case the message entity is backed by an input stream of which no byte has been read yet" ?